packages/web-components/examples/codesandbox/components/content-block-media/src/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-web-components/es/components/content-block-media/content-block-media';
+import '@carbon/ibmdotcom-web-components/es/components/content-block-media/content-block-media-content';
+import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-heading';
+import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-paragraph';
+import '@carbon/ibmdotcom-web-components/es/components/content-group/content-group-heading';
+import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-heading';
+import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-copy';
+import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item';
```

Comment:
Some bundlers, including Snowpack, require `.js` extension:

```suggestion
import '@carbon/ibmdotcom-web-components/es/components/content-block-media/content-block-media.js';
import '@carbon/ibmdotcom-web-components/es/components/content-block-media/content-block-media-content.js';
import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-heading.js';
import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-paragraph.js';
import '@carbon/ibmdotcom-web-components/es/components/content-group/content-group-heading.js';
import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-heading.js';
import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-copy.js';
import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item.js';
```